erpc_c/transports/dspi_slave_transport.cpp

```diff
@@ -57,11 +57,16 @@ DspiSlaveTransport::DspiSlaveTransport(SPI_Type *spiBaseAddr, uint32_t baudRate,
 : m_spiBaseAddr(spiBaseAddr)
 , m_baudRate(baudRate)
 , m_srcClock_Hz(srcClock_Hz)
+, m_binited(false)
 {
 }
 
 DspiSlaveTransport::~DspiSlaveTransport()
 {
+    if(m_binited)
```

Comment:
Please add a space between the "if" and "(" to match our coding style (same coding style as KSDK).